### PR TITLE
Add Koha

### LINKED ISSUE #2048: Detect Koha (library management system)
This is a preparation for adding the detection of Koha.

#### Front office, called [OPAC](https://en.wikipedia.org/wiki/Online_public_access_catalog)

`<meta name="generator" content="Koha 17.1102000">`

We can even have the version.

#### Back office
Haven't yet found anything better than this:
`<input name="koha_login_context" value="intranet" type="hidden">` 
And it's for the login page.

##### List of libraries that publicly state that they use Koha through a stats module
https://hea.koha-community.org/libraries
(a lot don't think about or don't want to share stats)

##### Complementary info about the number of libraries
4832 declared in another source https://librarytechnology.org/lwc-processquery.pl?ILS=Koha
Found 15000 for the total estimate.
###### Sources
https://lists.katipo.co.nz/public/koha/2016-August/046089.html
> it is likely that Koha is the most implemented ILS globally

Wow this ↑ is amazing, I actually didn't know that :D

https://www.catalyst-eu.net/news/koha-introduction-courses

##### Wikipedia article
https://en.wikipedia.org/wiki/Koha_(software)

### PR DESCRIPTION
**Pull request by jvoisin**
This should close #2048

### CODE REVIEW

**Comment by gadcam** [comment]
You can also add the js pattern <code>KOHA</code> if you want cf http://koha.adminkuhn.ch/

File: src/apps.json [hunk 1]
```diff
@@ -4982,6 +4982,24 @@
...
     },
+    "Koha": {
+      "cats": [
+        21
+      ],
+      "meta": {
+        "generator": "^Koha ([\\d.]+)$\\;version:\\1"
+      },
+      "js": {
+        "KOHA": ""
```
Comment (tuxayo) [inline src/apps.json]: Even after looking at the [doc](https://www.wappalyzer.com/docs/specification) I don't get what this pattern does. (how JS pattern work actually)

Comment (gadcam) [reply src/apps.json]: The regexes are matched against the content of the Javascript methods (if necessary cf https://developer.mozilla.org/en-US/docs/Glossary/Method for what is a method in JS)
So in this case as we have no regex we just check if the object <code>KOHA</code> does exist.

Comment (tuxayo) [reply src/apps.json]: Thanks, I get it. Indeed there is a KOHA object in the OPAC and the back office, even in the login, page. This is great!

**Pull request merged**